vernac/g_vernac.mlg

```diff
@@ -566,7 +576,7 @@ GRAMMAR EXTEND Gram
 
       (* Requiring an already compiled module *)
       | IDENT "Require"; export = export_token; qidl = LIST1 global ->
-          { VernacRequire (None, export, qidl) }
+          { check_for_omega qidl; VernacRequire (None, export, qidl) }
```

Comment:
Please no!

Reply:
I'm removing this--but just to clarify, why the exclamation point?  Deprecating a module seems, in general, like a reasonable approach.

Reply:
I do not mind a file being deprecated. But I would veto any try at hardcoding such a deprecation warning directly in the parser rule for `Require`.